examples/app-basic/src/pages/PageFour/PageFour.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { LinkButton, useStyles2 } from '@grafana/ui';
+import { PLUGIN_BASE_URL, ROUTES } from '../../constants';
+
+export const PageFour = () => {
+  const s = useStyles2(getStyles);
+
+  return (
+    <div className={s.page}>
+      <div className={s.container}>
+        <LinkButton icon="arrow-left" href={`${PLUGIN_BASE_URL}/${ROUTES.One}`}>
```

Comment:
nit: maybe worth adding a utility function to create the route.

```ts
function routeTo(route: ROUTES): string {
   return `${PLUGIN_BASE_URL}/${route}`;
}
```

Reply:
Fixed in https://github.com/grafana/grafana-plugin-examples/pull/26/commits/c256a1c6086781c73314b8534ea6bee06fa84b29.